src/Analysis/Codelyzer.Analysis.Build/ProjectBuildHandler.cs

```diff
@@ -101,7 +103,38 @@ private void FallbackCompilation()
                 Compilation = CSharpCompilation.Create(Project.AssemblyName, trees, meta, options);
             }
         }
+        private async Task SetSyntaxCompilation()
```

Comment:
This method declaration is throwing a check warning in GH because it always runs synchronously (no `await` operator is called). Can replace `async Task` with `void` unless the one of the statements in here should be called with `await`.

Reply:
Ok